libsolidity/interface/CompilerStack.cpp

```diff
@@ -561,11 +558,8 @@ StringMap CompilerStack::loadMissingSources(SourceUnit const& _ast, std::string
 				newSources[importPath] = result.contentsOrErrorMessage;
 			else
 			{
-				auto err = make_shared<Error>(Error::Type::ParserError);
-				*err <<
-					errinfo_sourceLocation(import->location()) <<
-					errinfo_comment("Source \"" + importPath + "\" not found: " + result.contentsOrErrorMessage);
-				m_errors.push_back(std::move(err));
+				m_errorReporter.error(Error::Type::ParserError, import->location(), 
```

Comment:
Can you please format it as
```
m_errorReport.error(
	Error::Type::ParserError,
	import->location()
	string("...")
);
```

This applies to all the others in order to keep a consistent style.
```